Makefile

```diff
@@ -408,19 +416,30 @@ clean-all:
 # TODO: use env variables for container versions
 #<<<
 clean-%:
-	# # Clean public using native makefile
-	# if [[ $* == public ]]; then \
-	# 	cd lib/bento_public && $(MAKE) clean-public ; \
-	# 	exit \
-	# fi &>> tmp/logs/${EXECUTED_NOW}/$*/clean.log
-
-	@mkdir -p tmp/logs/${EXECUTED_NOW}/$* && \
-		echo "-- Stopping $* --" && \
-		docker-compose stop $* &> tmp/logs/${EXECUTED_NOW}/$*/clean.log
+	@mkdir -p tmp/logs/${EXECUTED_NOW}/$*
 	
 	@echo "-- Removing bentov2-$* container --" && \
 		docker rm bentov2-$* --force >> tmp/logs/${EXECUTED_NOW}/$*/clean.log 2>&1
 	
+	@# Clean public using native makefile
+	@if [[ $* == public ]]; then \
+		cd lib/bento_public && $(MAKE) clean-public ; \
+	fi &>> tmp/logs/${EXECUTED_NOW}/$*/clean.log
+
+	@# Clean gohan using native makefile
+	@if [[ $* == gohan ]]; then \
+		cd lib/gohan &&  \
+		$(MAKE) clean-api ; \
+	fi &>> tmp/logs/${EXECUTED_NOW}/$*/clean.log
+
+
+	@# Skip triggering top level makefile stop for both public and gohan
+	@if [[ $* != public && $* != gohan ]]; then \
+		echo "-- Stopping $* --" ; \
+		docker-compose stop $* &> tmp/logs/${EXECUTED_NOW}/$*/clean.log ; \
+	fi &>> tmp/logs/${EXECUTED_NOW}/$*/clean.log
```

Comment:
This would break on MacOS

Reply:
because of the `&>>` ?

Reply:
Yes. We have patched this by `>>` `2>&1` in all the places

Reply:
as this is not critical to the impending deployment in production, I'd say we elaborate on this next round on `qa/2.6`